log/log_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
The log package is licensed under whatever upstream is (pls check the other file headers). Since it's not our code, only a derivative, there's no reason to add out changes as something different.

Reply:
They don't display license in file headers. It's Apache 2.0 though. Should I just nuke this?

Reply:
Then yes

Reply:
Hm, I might just nuke this file. I removed the old escape function, so this file is useless now